### DISCUSSION TITLE
[Kotlin] Add some information of BotBus to ClientBus in botApi Mode

### DISCUSSION THREAD

**Discussion by djowalker** [theopenconversationkit/tock#1438] [Show and tell]
A little Sequence Diagram of what we use yet to retrieve action list of the current dialog in a bot application with botApi mode.
![drawio-BOTBUS to clientBus drawio](https://user-images.githubusercontent.com/22080627/214639116-4b37b68e-6425-4535-aa93-cad5864dcff1.png)

The idea is to use the UserContext to transmit some more informations about the dialog or other things.

And in a story of a botApplication you can use.

```
this as TockClientBus

this.request.context.NEWVARIABLE
```

**Comment by kfaber33** [comment]
Hi djowalker, it seems you know the tock platform very well! we are a Swiss IT company and we are looking for a professional that can help us in using tock for our customers, doing implementations and customizations. Are you available?

**Reply by djowalker** [reply]
Hello @kfaber33, thank you for your interest, however i am not available.
